io/plugins/eu.esdihumboldt.hale.io.codelist.skos.test/src/data/test1.rdf

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:skos="http://www.w3.org/2004/02/skos/core#" xmlns:dc="http://purl.org/dc/elements/1.1/">
+	<skos:Concept rdf:about="http://vocab.to.wetransform/term/W080/1/CONT0006">
```

Comment:
Ok, so it is not unlikely that the usage of `skos:concept` in other places is incorrect, and we only support it for compatibility reasons.